### PR TITLE
Read content config generation

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Comment by bratseth** [comment]
Waiting for https://github.com/vespa-engine/vespa/pull/26107 to merge

File: container-core/pom.xml [hunk 1]
```diff
@@ -453,6 +453,8 @@
             <arg>-Xlint:-try</arg>
             <arg>-Werror</arg>
           </compilerArgs>
+            <source>17</source>
```
Comment (freva) [inline container-core/pom.xml:456]: Why is this needed?

**Review by freva — APPROVED** [review]

File: container-core/pom.xml (hunk 1 above)
Comment (bratseth) [reply container-core/pom.xml]: Sorry, it's not, just more IntelliJ inserts - deleted.
